internal/users/add.go

```diff
@@ -74,3 +82,15 @@ func (c *Client) Add() *cobra.Command {
 
 	return addUserCmd
 }
+
+func validateRoles(roles []string) ([]metal.InvitationRolesInner, error) {
+	validRoles := make([]metal.InvitationRolesInner, len(roles))
+	for _, role := range roles {
+		validRole, err := metal.NewInvitationRolesInnerFromValue(role)
+		if err != nil {
+			return nil, err
```

Comment:
Again opted to return the original error without wrapping it, since the message looks good to me: https://github.com/equinix-labs/metal-go/blob/main/metal/v1/model_invitation_roles_inner.go#L62